jetty-servlet/src/test/java/org/eclipse/jetty/servlet/ServletRequestLogTest.java

```diff
@@ -64,6 +39,29 @@
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
 
+import javax.servlet.AsyncContext;
+import javax.servlet.AsyncEvent;
+import javax.servlet.AsyncListener;
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.time.Duration.ofSeconds;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
```

Comment:
The reorganization of the imports is bothersome.

This points to not using the style guide.

1. Static Imports
2. `java.*`
3. `javax.*`
4. Everything else.

Reply:
Dang DumbJ??? I've imported the xml that you have pinned (twice) and this is still the ordering that it gives me.   I actually like this ordering better.... but if the static imports first is more the norm then I'm happy to go with that, but no idea (pun intended) why the IDE is not doing that?